File: viviane013/programacao_basica/Questao08.java
import java.util.Scanner;
public class Questao08 {
        public static void main(String[] args) {
            Scanner s;
            int altura;
            int comprimento;
            int largura;
            int calculo;
            int alt;
            int larg;
            int calc;
            int comp;
            int al;
            int cal;
            int esfera;
            int raio;
            int divi;

            s = new Scanner(System.in);

            System.out.println("Contador de volume das formas");
            
            System.out.print("informe o numero para a altura do paralelepipedo: ");
            altura = s.nextInt();
            System.out.print("Informe o numero para o comprimento: ");
            comprimento = s.nextInt();
            System.out.print("Informe o numero para a largura: ");
            largura = s.nextInt();

            calculo = altura * comprimento * largura;
            System.out.println("O volume de um paralelepipedo eh: " + calculo);

            System.out.print("Informe o numero para o lado do cubo: ");
            alt = s.nextInt();
            System.out.print("Informe o numero para largura: ");
            larg = s.nextInt();
            
            calc = alt * larg;
            System.out.println("O lado de um cubo eh:" + calc);

            System.out.print("Informe o numero para o volume do cone: ");
            comp = s.nextInt();
            System.out.print("Informe a altura: ");
            al = s.nextInt();

            cal = comp * al;
            System.out.println("O volume do cone eh " + cal);
            
            System.out.print("Informe o numero para a esfera: ");
            esfera = s.nextInt();
            System.out.print("Informe o para encontrar o raio: ");
            raio = s.nextInt();
            
            divi = esfera / raio;
            System.out.println("o raio da esfera eh: " + divi);
            
            
            


            

    }
}

File: jakyson024/programacao_basica/questao08.py

x = float(input("Informe o lado do cubo: ")) 

n = float(input("Informe a altura do paralelepipedo: "))
u = float(input("Informe o comprimento do paralelepipedo: "))
s = float(input("Informe o profundidade do paralelepipedo: "))

q = float(input("Informe a base do oone: "))
w = float(input("Informe a altura do cone: "))


e = float(input("Infome o raio da esfera: "))

r = x*x*x

t = n*u*s


y = (3.14 * (q/2)**2 * w)/3

i =  4/3 * 3.14 * e**3

print("Volume do cubo: " + str(r))
print("Volume do paralelepipedo: " + str(t))
print("Volume do cone: " + str(y))
print("Volume da esfera: " + str(i))

 
